documentation/source/library_reference.rst

```diff
@@ -166,6 +166,21 @@ Simulation Object Handles
     :show-inheritance:
     :synopsis: Classes for simulation objects.
 
+.. _set-actions-section
```

Comment:
Could change this to `assignment-methods-section` to match the heading. Fine to leave as is if you think matching the python class name is more important.